application/auth/views.py

```diff
@@ -84,3 +85,10 @@ def reset_password(token):
         return render_template("auth/password_updated.html", form=form, token=token, user=user)
 
     return render_template("auth/reset_password.html", form=form, token=token, user=user)
+
+
+@auth_blueprint.route("/logout", methods=["POST"])
+def logout():
+    session.clear()
+    logout_user()
```

Comment:
Is there a benefit to using `flask_login`'s `logout_user` as opposed to redirecting to `flask_security`'s `logout` view?

It feels like it is probably better to go through `flask_security`, as that's what we're using to login. In which case, it could be like this (inline or not):

```
@auth_blueprint.route("/logout", methods=["POST"])
def logout():
    session.clear()
    from flask_security.views import logout as security_logout_view

    return security_logout_view()
```

Reply:
Probably makes sense to stick with the Flask Security one I guess to match login. I'm not sure what the difference is, and suspect that one probably calls the other under the hood.

Will change now.

Reply:
Yep, looks like the Flask Security one calls the flask_login one eventually I think.

Reply:
Yeah it does, but it also does some extra stuff like clearing some other cookies, I think.